extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -574,7 +574,7 @@ export class AmpStoryPage extends AMP.BaseElement {
             this.cssVariablesStyleEl_.setAttribute('type', 'text/css');
             doc.head.appendChild(this.cssVariablesStyleEl_);
           }
-          this.cssVariablesStyleEl_.textContent = `:root {--story-page-vh: ${px(
+          this.cssVariablesStyleEl_.textContent = `:root {--story-dvh: ${px(
```

Comment:
Let's update this to directly set the property on `document.documentElement`

Reply:
Also, shouldn't we measure the viewport size here instead (sth like `document.getCLientBoundingBox().height`)?
And do that maybe on `amp-story.js` instead of on all pages?